srv/components/misc_pkgs/kafka/stop.sls

```diff
@@ -30,9 +28,5 @@ Wait for kafka to stop:
       - Stop kafka
 
 Stop zookeeper:
-  cmd.run:
-    - name: ./bin/zookeeper-server-stop.sh -daemon config/zookeeper.properties
-    - cwd: /opt/kafka
-    - onlyif: test 1 -le $(ps ax | grep java | grep -i QuorumPeerMain | grep -v grep | awk '{print $1}' | wc -l)
-    - require:
-      - Wait for kafka to stop
+  service.dead:
```

Comment:
Don't delete
Maintain the commands commented, just in case we have to fallback.